### DISCUSSION TITLE
Welcome to Metamong Report board!!

### DISCUSSION THREAD

**Discussion by minyuyeong** [metamong-friends/metamong-friends.github.io#2] [공지사항]
Metamong을 사용해주셔서 감사합니다!

사용 중 기능에 대해 궁금한 점은 >>문의하기<<
사용 중 발견하신 버그 제보는 >>제보하기<<
메타몽에 추가되면 좋을 새로운 기능에 대한 아이디어는 >>제안하기<< 게시판에 등록해주세요:)

**Comment by ACT-TJ** [comment]
커맨트 샘플
